app/src/main/java/com/myperishableplanner/v21001/ItemViewModel.kt

```diff
@@ -40,9 +43,9 @@ class ItemViewModel (var itemService: IItemService = ItemService()): ViewModel()
             firestore.collection("itemDetail").document(itemDetail.itemDetailId)
         }
         itemDetail.itemDetailId = document.id
-        document.set (itemDetail)
-        val handle = document.set (itemDetail)
+
+        val handle = document.set(itemDetail)
```

Comment:
redundant call to document.set(itemDetail)

Reply:
Good.